pennylane/transforms/defer_measurements.py

```diff
@@ -226,8 +226,7 @@ def func(x, y):
     _check_tape_validity(tape)
 
     device = kwargs.get("device", None)
-
-    device = kwargs.get("device", None)
+    reduce_postselected = kwargs.get("reduce_postselected", True)
```

Comment:
Can you add this kwarg to the `defer_measurements` docstring?

Reply:
Done! Added it as explicit kwarg as well, rather than extracting it from `kwargs`. I hope that works?